waldo_server/server.js

```diff
@@ -66,13 +67,34 @@ app.post('/photo', (req, res) => {
           console.log(`YAYYY SAVED!`);
         }
       });
-      res.send('Okay')
+      res.send('Okay');
     } else {
       res.send('Error');
     }
   });
 });
 
+app.put('/photo/:id', (req, res) => {
```

Comment:
wanted to note that for route naming practices, we generally want to go plural instead singular for most routes. technically it's up to opinion and there are a ton of articles about this topic, but most if not all REST endpoints i've seen and consumed prefer plural in the wide majority of cases. eg:

```
GET /photos
GET /photos/:id
POST /photos
PUT /photos/:id
PATCH /photos/:id
DELETE /photos/:id
```

of course there will be exceptions to the norm, but there aren't any of those in this file yet.